{
  "Faults": [
    {
      "latitude": 51.405257331992395,
      "type": "LV",
      "reference": "MV5794",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-11-16T09:30:00.000+0000",
      "affectedCustomerCount": 2,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "b962b0f7-6c3b-47a4-ba3c-b3900fd2ff81",
      "minLongitude": -0.8478,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8335355827685007,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-16T07:00:00.000+0000",
      "estimatedRestoration": "2023-11-17T03:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MV5794'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MV5794",
      "maxLongitude": -0.8081,
      "minLatitude": 51.4022,
      "name": "RG41 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG40 2JU",
        "RG41 2QZ",
        "RG41 2RF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8478,
              51.4072
            ],
            [
              -0.8081,
              51.4072
            ],
            [
              -0.8081,
              51.4022
            ],
            [
              -0.8478,
              51.4022
            ],
            [
              -0.8478,
              51.4072
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-11-16T09:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4072,
      "etrUpdate": true
    },
    {
      "latitude": 51.4930249806004,
      "type": "HV",
      "reference": "MV6910",
      "etaUpdate": true,
      "affectedCustomerCount": 1,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "b3a3aa3e-86b4-42db-8f05-67175fc44244",
      "minLongitude": -0.3711,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.3273974012452433,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-16T18:07:00.000+0000",
      "estimatedRestoration": "2023-11-16T23:59:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MV6910'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MV6910",
      "maxLongitude": -0.3013,
      "minLatitude": 51.4826,
      "name": "UB5 & TW8 Areas",
      "otherUpdate": true,
      "affectedAreas": [
        "TW7 5PR",
        "TW7 5QD",
        "TW8 9BQ",
        "TW8 9DD",
        "TW8 9DE",
        "TW8 9DJ",
        "TW8 9DN",
        "TW8 9DU",
        "TW8 9EX",
        "TW8 9ZB",
        "UB5 6AG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.3711,
              51.5458
            ],
            [
              -0.3013,
              51.5458
            ],
            [
              -0.3013,
              51.4826
            ],
            [
              -0.3711,
              51.4826
            ],
            [
              -0.3711,
              51.5458
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-11-16T18:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5458,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-16T16:50:00.000+0000",
      "latitude": 51.58881443987422,
      "estimatedRestoration": "2023-11-17T01:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MV6688'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MV6688",
      "reference": "MV6688",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-11-16T17:45:00.000+0000",
      "name": "UB9 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "UB9 6PE"
      ],
      "location": {
        "coordinates": [
          -0.4818062196283084,
          51.58881443987422
        ],
        "type": "Point"
      },
      "UUID": "e3360b7a-c4c4-45e3-a302-a380cd857038",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -0.4818062196283084,
      "resolved": false
    },
    {
      "latitude": 51.4028708322169,
      "type": "LV",
      "reference": "MV6584",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-11-16T17:50:00.000+0000",
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "e70bb479-eacc-43d9-a0d6-9fad67c3d8d5",
      "minLongitude": -0.9475,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.9461189245861038,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-16T15:38:00.000+0000",
      "estimatedRestoration": "2023-11-17T05:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MV6584'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MV6584",
      "maxLongitude": -0.94,
      "minLatitude": 51.399,
      "name": "RG2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG2 9EN"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.9475,
              51.4036
            ],
            [
              -0.94,
              51.4036
            ],
            [
              -0.94,
              51.399
            ],
            [
              -0.9475,
              51.399
            ],
            [
              -0.9475,
              51.4036
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4036,
      "etrUpdate": true
    },
    {
      "latitude": 50.745850764287205,
      "type": "LV",
      "reference": "MV7118",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-11-17T02:20:00.000+0000",
      "affectedCustomerCount": 21,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "448804ea-b1f5-46a5-a5f5-58c6d48b9860",
      "minLongitude": -2.0207,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.0122082452617205,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-17T00:59:00.000+0000",
      "estimatedRestoration": "2023-11-17T06:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MV7118'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MV7118",
      "maxLongitude": -2.0085,
      "minLatitude": 50.7378,
      "name": "BH17 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "BH16 5SR",
        "BH17 7YR",
        "BH17 7YS",
        "BH17 7YT"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.0207,
              50.7492
            ],
            [
              -2.0085,
              50.7492
            ],
            [
              -2.0085,
              50.7378
            ],
            [
              -2.0207,
              50.7378
            ],
            [
              -2.0207,
              50.7492
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-11-17T02:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.7492,
      "etrUpdate": true
    },
    {
      "latitude": 51.456509100186146,
      "type": "LV",
      "reference": "MV7124",
      "etaUpdate": true,
      "affectedCustomerCount": 16,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "8f657e36-5ef2-47dc-afe2-16f512059ca1",
      "minLongitude": -0.8954,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8878735919329654,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-17T01:20:00.000+0000",
      "estimatedRestoration": "2023-11-17T05:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MV7124'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MV7124",
      "maxLongitude": -0.884,
      "minLatitude": 51.454,
      "name": "RG5 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "RG5 4RY",
        "RG5 4SE",
        "RG5 4SH",
        "RG5 4SJ",
        "RG5 4XL"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8954,
              51.4618
            ],
            [
              -0.884,
              51.4618
            ],
            [
              -0.884,
              51.454
            ],
            [
              -0.8954,
              51.454
            ],
            [
              -0.8954,
              51.4618
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2023-11-17T02:20:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4618,
      "etrUpdate": true
    },
    {
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2023-11-16T01:25:00.000+0000",
      "latitude": 51.3444011681348,
      "estimatedRestoration": "2023-11-17T10:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'MV7078'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/MV7078",
      "reference": "MV7078",
      "etaUpdate": true,
      "engineerOnSiteTime": "2023-11-16T23:45:00.000+0000",
      "affectedCustomerCount": 2,
      "name": "SN8 Area",
      "otherUpdate": true,
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SN8 3BU",
        "SN8 3BX"
      ],
      "location": {
        "coordinates": [
          -1.6578586333292216,
          51.3444011681348
        ],
        "type": "Point"
      },
      "UUID": "f824515c-bb0f-49cb-ae7e-4a84c60449fe",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "etrUpdate": true,
      "longitude": -1.6578586333292216,
      "resolved": false
    }
  ]
}
